README.md

```diff
@@ -344,5 +344,5 @@ Install via Nuget, follow these
 [releases]: https://github.com/GoogleCloudPlatform/cloudsql-proxy/releases
 [roles-and-permissions]: https://cloud.google.com/sql/docs/mysql/roles-and-permissions
 [service-account]: https://cloud.google.com/iam/docs/service-accounts
-[sidecar-example]: https://github.com/GoogleCloudPlatform/cloudsql-proxy/tree/master/examples/kubernetes#run-the-cloud-sql-proxy-as-a-sidecar
+[sidecar-example]: https://github.com/GoogleCloudPlatform/cloudsql-proxy/tree/master/examples/sidecar#run-the-cloud-sql-proxy-as-a-sidecar
```

Comment:
Should probably be either `kubernetes/sidecar` or `k8s-service`

Reply:
Will do. I like the shallow option: `k8s-service` and `k8s-sidecar`.